programs/token/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "mollusk-token"
+version = "0.1.0"
+edition = "2021"
```

Comment:
Same here on the manifest configuration.
https://github.com/buffalojoec/mollusk/blob/6810892b26ac39feebe441c5f3d2daa8f6ba6735/harness/Cargo.toml#L3-L10